functional/provision/setup.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
```

Comment:
nevermind my last comment; did a complete move now anyways as what might make sense in a setup.sh might need privileges which we donÄ't need for install_go.sh and if we ever needed it we can run extra setup.sh